app/helpers/responses_helper.rb

```diff
@@ -49,6 +49,12 @@ def get_form_field answer_form
     content_tag(field_type, '', attributes)
   end
 
+  def is_mandatory question
+    question.is_a?(Question) ||
+      question.is_a?(CustomisedQuestion) ||
+      question.is_a?(DemographicQuestion) && question.validation == {required: true}.to_json
```

Comment:
@WKocur this PR is great! I just want to check about the indentation here. I am not familiar with this kind of indentation that doesn't mean it is wrong I just wanted to check!